fs/file.c

```diff
@@ -957,6 +957,7 @@ int f_dupfd(unsigned int from, struct file *file, unsigned flags)
 	}
 	return err;
 }
+EXPORT_SYMBOL(f_dupfd);
```

Comment:
These three will probably require strong justification for each.

Reply:
rpmsg-hostfs need call these functions to fully redirect all remote side file related functions, but if we don't make rpmsg-hostfs as module, these function don't expose. How about we change tristate to bool in Kconfig?

Reply:
I'm not sure, this would be upto the clock maintainer and not me. Best to send the patches to LKML after you have made updates from this review.